workspaces/global-floating-action-button/plugins/global-floating-action-button/src/components/FABWithSubmenu.tsx

```diff
@@ -43,36 +55,57 @@ export const FABWithSubmenu = ({
   }, [pathname]);
 
   const handleClick = () => {
-    setIsMenuOpen(!isMenuOpen);
+    setIsMenuOpen(prev => !prev);
   };
   return (
     <>
-      <Tooltip title="Menu" placement="left">
+      <Tooltip
+        title="Menu"
+        placement={slot === Slot.PAGE_END ? 'left' : 'right'}
```

Comment:
Should we maybe somewhere have a "Slot to X" configuration option like this.

```tsx
const slotOptions = {
  [Slot.PAGE_START]: {
    tooltipDirection: "right",
  },
  [Slot.PAGE_END]: {
    tooltipDirection: "left",
  },
} as const;
```

So that we can use it here like:

```suggestion
        placement={slotOptions[slot].tooltipDirection}
```